packages/coding-standard/src/Rules/CheckUsedNamespacedNameOnClassNodeRule.php

```diff
@@ -77,6 +83,34 @@ public function process(Node $node, Scope $scope): array
         return [self::ERROR_MESSAGE];
     }
 
+    private function isVariableNamedShortClassName(Variable $variable): bool
+    {
+        $assign = $variable->getAttribute(PHPStanAttributeKey::PARENT);
+        while ($assign) {
+            if ($assign instanceof Assign) {
+                break;
+            }
+
+            $assign = $assign->getAttribute(PHPStanAttributeKey::PARENT);
+        }
+
+        if (! $assign instanceof Assign) {
+            return false;
+        }
```

Comment:
This could be extracted to parent method to `AbstractSymplifyRule`:

```php
/** @var Assign|null $assign */
$assig =$this->getFirstParentByType($node, Assign::class);
```

Reply:
implemented.